bsb/morphologies/__init__.py

```diff
@@ -930,17 +930,21 @@ def simplify(self, *args, optimize=True, **kwargs):
     @classmethod
     def from_swc(cls, file, branch_class=None, tags=None, meta=None):
         """
-        Create a Morphology from a file-like object.
+        Create a Morphology from an SWC file.
```

Comment:
```suggestion
        Create a Morphology from an SWC file or file-like object.
```

This still accepts both file paths and file-likes